src/CONST.ts

```diff
@@ -3036,6 +3039,15 @@ const CONST = {
         get EXPENSIFY_POLICY_DOMAIN_NAME() {
             return new RegExp(`${EXPENSIFY_POLICY_DOMAIN}([a-zA-Z0-9]+)\\${EXPENSIFY_POLICY_DOMAIN_EXTENSION}`);
         },
+
+        /**
+         * Matching task rule by group
+         * Group 1: Start task rule with []
+         * Group 2: Optional email group between \s+....\s* start rule with @+valid email or short mention
+         * Group 3: Title is remaining characters
+         */
+        // The regex is copied from the expensify-common CONST file, but the domain is optional to accept short mention
```

Comment:
```suggestion
```

I know we just moved this, but I think we can remove this comment. It doesn't seem relevant.